### PR TITLE
Componentized the Codebase

### PR DESCRIPTION
**Pull request by harshkhandeparkar**
- Moved page sections into `sections/` from `components/`
- Created `Title` component
- Added `Title` and most styles directly to the `Section` component

### CODE REVIEW

**Comment by rajivharlalka** [comment]
image
Looks the UI broke

**Comment by harshkhandeparkar** [comment]
Fixed.

**Comment by harshkhandeparkar** [comment]
> image bottom is less

It is supposed to get fixed when the footer is added.

File: src/components/Section.astro [hunk 1]
```diff
@@ -24,10 +47,91 @@ const { id } = Astro.props as Props;
...
+      font-size: 4cqi;
+    }
+
+    .section-description {
+      font-size: 4cqi;
+    }
+  }
 </style>
+
+<style lang="scss"></style>
```
Comment (rajivharlalka) [inline src/components/Section.astro:137]: please remove this

File: src/components/Section.astro [hunk 2]
```diff
@@ -1,21 +1,44 @@
 ---
+import Title from "./Title.astro";
+
 export interface Props {
   id: string;
+  title?: string;
+  titleAlignment?: string;
```
Comment (rajivharlalka) [inline src/components/Section.astro:7]: we dont need this

File: src/sections/UpcomingEvents.astro
Comment (rajivharlalka) [inline src/sections/UpcomingEvents.astro:1]: image
let's fix this

**Review by rajivharlalka — CHANGES_REQUESTED** [review]

**Review by rajivharlalka — APPROVED** [review]

**Pull request merged**